app/src/main/java/io/homeassistant/companion/android/vehicle/MainVehicleScreen.kt

```diff
@@ -62,6 +62,11 @@ class MainVehicleScreen(
             "sensor",
             "zone"
         )
+
+        val NOT_ACTIONABLE_DOMAINS = listOf(
```

Comment:
As you created a `DomainCheck` file, maybe it is a good idea to make that a vehicle supported entities file and move this constant and the other existing ones above to it so everything is in one place?

Reply:
done, I opted to keep the file the same as everything is still related to the `domain`